db/schema.rb

```diff
@@ -103,9 +103,9 @@
     t.string "unlock_token"
     t.datetime "locked_at"
     t.string "true_layer_secure_data_id"
-    t.boolean "employed"
```

Comment:
@jsugarman - can we drop the commit with the schema changes? I am not sure how they have crept it but I don't think they are required for this ticket.